purchases/src/customEntitlementComputation/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -13,7 +13,6 @@ open class PurchasesConfiguration(builder: Builder) {
     internal val store: Store
     internal val diagnosticsEnabled: Boolean
     val dangerousSettings: DangerousSettings
-    val verificationMode: EntitlementVerificationMode
```

Comment:
this was still available in this mode.

Reply:
Ah right... In iOS, we don't configure the SDK by passing a `PurchasesConfiguration`... Honestly, that feels less flexible (in case we ever do want to add optional features in the future) but it does disallow enabling these features... IMO, I think it's better to keep it this way for future-proofing, but lmk if you prefer the other way